unittests/test_import_reimport.py

```diff
@@ -1143,6 +1145,35 @@ def test_import_param_close_old_findings_with_additional_endpoint(self):
         engagement_findings_count = Finding.objects.filter(test__engagement_id=1, test__test_type=test.test_type, active=True, is_mitigated=False).count()
         self.assertEqual(engagement_findings_count, 0)
 
+    def test_import_reimport_generic(self):
```

Comment:
Should this test check, if the files are processed properly or is this not necessary?

Reply:
This test use a part of the code that were not covered before (having FileUpload in a finding).
This part fail without the patch of this PR

Reply:
This unit test failing shows that: https://github.com/DefectDojo/django-DefectDojo/pull/5632/commits/ed13a02c3ab289b8444dfb04e5359b55b9fe5119

Reply:
Great, just wanted to make sure that files are covered.